pkgs/development/python-modules/devgoldyutils/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  pname = "devgoldyutils";
+  version = "3.0.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "2kGu9QPP5WqKv2gO9DAkE9SNDerzNaEDRt5DrrYD9nQ=";
+  };
+
+  doCheck = false;
+
+  nativeBuildInputs = [
+    setuptools-scm
+  ];
+
+  pythonImportsCheck = [ "devgoldyutils" ];
+
+  meta = with lib; {
+    description = "A collection of utility functions for Python used by mov-cli";
+    homepage = "https://github.com/THEGOLDENPRO/devgoldyutils";
+    license = licenses.gpl3Only;
+    maintainers = with maintainers; [ roshaen ];
+  };
```

Comment:
#208242

Reply:
I'm little confused here
I observed this pattern is used in every other python-module
Can you please tell me what am I missing here?

Reply:
The issue linked above is an on-going effort to clean it up

You aren't missing anything - both approaches will work, it's just that reducing the occurrences of `with lib` at a block level is the preferred approach, so we're trying to avoid introducing new packages which would have to be cleaned up later